.github/workflows/integration_test.yaml

```diff
@@ -3,7 +3,7 @@
 #
 
 ---
-name: Run integration test
+name: Integration Test
```

Comment:
because the yaml file name changes, the badges in `README.md` needs to be updated too.